lib/internal/crypto/ec.js

```diff
@@ -77,14 +77,17 @@ function createECPublicKeyRaw(namedCurve, keyData) {
   return new PublicKeyObject(handle);
 }
 
-async function ecGenerateKey(algorithm, extractable, keyUsages) {
-  const { name, namedCurve } = algorithm;
-
-  if (!ArrayPrototypeIncludes(ObjectKeys(kNamedCurveAliases), namedCurve)) {
+function validateEcGenerateKeyAlgorithm(algorithm) {
```

Comment:
You can also use this in [`ecImportKey`](https://github.com/nodejs/node/blob/47a59bde2aadb3ad1b377c0ef12df7abc28840e9/lib/internal/crypto/ec.js#L161-L165).